Sources/active-win/main.swift

```diff
@@ -68,14 +88,22 @@ for window in windows {
 			"height": bounds.height
 		],
 		"owner": [
-			"name": window[kCGWindowOwnerName as String] as! String,
+			"name": appName,
 			"processId": appPid,
 			"bundleId": app.bundleIdentifier!,
 			"path": app.bundleURL!.path
 		],
 		"memoryUsage": window[kCGWindowMemoryUsage as String] as! Int
 	]
 
+	// Only run the AppleScript if active window is a compatible browser
+	if browserURLAppleScriptCommand != nil {
+		let browserURL = runAppleScript(source: browserURLAppleScriptCommand!)
+		if browserURL != nil {
+			dict["url"] = browserURL
+		}
+	}
```

Comment:
```swift
if
	let script = getActiveBrowserTabURLAppleScriptCommand(appName),
	let url = runAppleScript(source: script)
{
	dict["url"] = url
}
```

Reply:
Cool, didn't realize Swift had this functionality in the if conditional.